ansible/roles/windows/sshd/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Check if sshd service exists
```

Comment:
Since this was moved out from `windows/common/tasks/main.yml`, what do you think about moving it to `windows/common/tasks/sshd.yml`?

Reply:
I think it is better to keep it as a separate role, this way it will be easier to reuse it in other project as all constants are contained in its own defaults directory.

Reply:
Agreed.